packages/config/config/devices/0x0176/tzwp-102.json

```diff
@@ -93,6 +93,76 @@
 					"value": 0
 				}
 			]
+		},
+		{
+			"#": "17",
+			"label": "Report on change of power consumption (%)",
```

Comment:
We should distingish between param 16 and this one better. I suggest:
Param 16:
- change label to "Power Consumption Reports: Change Threshold"
- add unit `W`

Param 17:
- change label to "Power Consumption Reports: Relative Change Threshold"
- add unit `%`